apps/admin/src/atoms/useAuthAtom.ts

```diff
@@ -0,0 +1,51 @@
+import { LOCAL_STORAGE } from '@boolti/api';
+import { useAtom } from 'jotai';
+import { atomWithStorage, RESET } from 'jotai/utils';
+import { useMemo } from 'react';
+
+const storageMethod = {
+  getItem: (key: string, initialValue: string | null) => {
```

Comment:
null을 따로 쓴 이유가 있을까용?

```suggestion
  getItem: (key: string, initialValue?: string) => {
```

Reply:
스토리지에 없을 때 getItem 하면 null을 return하니까 null로 했음니다